x-pack/plugins/security_solution/public/management/pages/policy/store/policy_list/selectors.ts

```diff
@@ -84,3 +84,16 @@ export const urlSearchParams: (
 
   return searchParams;
 });
+
+/**
+ * Returns package information for Endpoint
+ * @param state
+ */
+export const endpointPackageInfo = (state: Immutable<PolicyListState>) => state.endpointPackageInfo;
+
+/**
+ * Returns the version number for the endpoint package.
+ */
+export const endpointPackageVersion = createSelector(endpointPackageInfo, (info) =>
+  info ? info.version : undefined
```

Comment:
could you use an optional or null coalesce here?

Reply:
yeah - I'll change it.